package-lock.json

```diff
@@ -5444,7 +5444,7 @@
         },
         "p-cancelable": {
           "version": "0.4.1",
-          "resolved": "http://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
+          "resolved": "https://registry.npmjs.org/p-cancelable/-/p-cancelable-0.4.1.tgz",
```

Comment:
I think a bunch of these `-lock` changes should not be checked in right? iirc I broke CI a while back by accidentally checking in similar changes

Reply:
I think having them go back to https should be fine. I still don't know why npm switches those occasionally other than that they seem to change betweel `npm install` and `npm add`. Regardless, I'll revert them since they're not related to what I meant to change.